src/tools/sfdk/command.cpp

```diff
@@ -79,9 +79,16 @@ class PropertiesAccessor
 {
     Q_DECLARE_TR_FUNCTIONS(Sfdk::PropertiesAccessor)
 
+public:
+    enum PrepareResult {
+        Ignored = 0,
+        Prepared,
+        Failed,
+    };
+
```

Comment:
Again I feel the commit message is misleading: All you did here was change the return value of prepareSet from simple bool to enum.

Reply:
It was really done in order to allow use of multiple properties accessors in a chain.

Reply:
A good commit message would explain _what_ was changed and _why_. You have only explained the _why_ part, and not _what_.

Reply:
More precisely it should be *what*, *why* and *how*. The commit title actually explains the *what*, not *why*. The *why* in this particular case is sufficiently explained by the fact that the commit precedes another commit(s) in a pull request. A good commit message should only try to explain *how* if the change is not easy to understand, which, according to your original comment, is not the case. I suggest to keep it as is.  :)

Reply:
No, it does not (explain _what_). The _what_ in this case would be "Change the type of prepareSet return value". _why_ is "... in order to allow to chain PropertiesAccessor results". I agree that _how_ in this case is easy enough to understand.

Reply:
- what: Allow to chain PropertiesAccessor results
- why: Because something later in the PR needs that (clear)
- how: Change the type of prepareSet return type... (clear)

Let's keep the bikes outside ;)

Reply:
Eh. Sorry, but the answer to the question _what was changed_ is really "The type of prepareSet return value".

Reply:
I wonder why is the title of this commit https://github.com/sailfishos/sailfish-qtcreator/pull/223/commits/023c3d57d0662073ea0830cbc97539992aa9d584 "Mer: Support targets on other build engines besides vbox", instead of something like "Fix MerSdkManager::kit by ...".